### PR TITLE
chore: replace non-breaking spaces

### PR DESCRIPTION
**Pull request by Conduitry**
A number of these have snuck in over the years. This adjusts the remaining instances that aren't synced from the Svelte or Kit repos.

### Before submitting the PR, please make sure you do the following

- [ ] It's really useful if your PR references an issue where it is discussed ahead of time.
- [x] Prefix your PR title with `feat:`, `fix:`, `chore:`, or `docs:`.
- [x] This message body should clearly illustrate what problems it solves.

### CODE REVIEW

File: apps/svelte.dev/src/routes/blog/+page.svelte [hunk 1]
```diff
@@ -81,7 +81,6 @@
 	article {
 		margin: 0 0 4rem 0;
 
-		/* we need to use :global because snippets don't currently cause a deopt */
```
Comment (Conduitry) [inline apps/svelte.dev/src/routes/blog/+page.svelte:84]: Random drive-by fix of an outdated comment.

**Review by dummdidumm — APPROVED** [review]
